### PR TITLE
Update BASE_URL for dev and prod

### PR DESCRIPTION
**Pull request by kalenluciano**
#### Fixes

* Update BASE_URL to be a env variable so the deployed dev and prod sites can point to their respective live APIs
* Change the publicPath so that the frontend has the base_url and the API is base_url/api

#### Description

This PR points the BASE_URL to an env variable so that we can point to the respective live APIs for dev and prod. The PR also changed the publicPath to be '/' so that the base_url can be the home page of the frontend

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,15 @@ npm install
 
 ```
 
-### 8. Run the development server.
+### 8. Create .env file and add the following env variables for the Vue app
+
+```
+VUE_APP_PDAP_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhcGlfa2V5IjoiNmVlOTQ2ZmI3YjY0NGUzNTgxNWVkMzMzOWY0MGRjNmQifQ.UMqlzKkJxwmz0bJy9BSONcoetg9zErl0XTkip1-W7a0
```
Comment (joshuagraber) [inline README.md:78]: Can we safely assume that committing this token publicly does not present a security risk?
And if so, why don't we just create a `.env.local` file or something and set package scripts to point to that in the test/local environment?

**Review by joshuagraber — DISMISSED** [review]
LGTM, just one security-related question on handling of ENV vars.

File: README.md (hunk 1 above)
Comment (mbodeantor) [reply README.md:78]: That's true, I think .env file with just this in it makes sense

Comment (kalenluciano) [reply README.md]: Makes sense to me, too

**Comment by mbodeantor** [comment]
@kalenluciano Looks like the .env file ended up in there despite the gitignore

**Comment by kalenluciano** [comment]
This is the .env file with the variables that are okay to be publicly facing. I see that it's causing GitHub to flag it as an error though. I was thinking we could just add them to a globals.js file instead, but we need the values to be env variables for deploying to Digital Ocean

**Review by mbodeantor — APPROVED** [review]
appreciate all the work on this

**Pull request merged**